database/migrations/2020_11_03_041734_create_questions_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateQuestionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('questions', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->integer('user_id');
```

Comment:
Bảng users, cột id cũng đang để là `bigIncrements` nên cột reference key cũng nên có cùng kiểu data 
```suggestion
            $table->unsignedBigInteger('user_id');
```